app/vmalert/web.go

```diff
@@ -67,6 +68,24 @@ func (rh *requestHandler) handler(w http.ResponseWriter, r *http.Request) bool {
 	}
 
 	switch r.URL.Path {
+	case "health":
```

Comment:
this endpoint can be abused by user.
I'd prefer the health check to happen in background once in an interval. The result of the check should be cached. The web UI should provide access to cached values only to prevent abuse.